app.js

```diff
@@ -0,0 +1,137 @@
+// Usage:
+// node app.js -r -c config.yaml -u "http://localhost:9001" # remember to add the registration!
+// node app.js -p 9001 -c config.yaml
+var qs = require('querystring');
+var requestLib = require("request");
+var yaml = require("yamljs");
+var bridge;
+
+function startServer(config, rooms, callback) {
+    if ("tls" in config) {
+        var fs = require("fs");
+        var tls_options = {
+            key: fs.readFileSync(config["tls"]["key_file"]),
+            cert: fs.readFileSync(config["tls"]["crt_file"])
+        };
+        var createServer = function(cb) { return require("https").createServer(tls_options, cb); };
+    } else {
+        var createServer = require("http").createServer;
```

Comment:
Done

(I was relying on the hoisting to make them the same variable, but I guess it is more clear to be explicit about that :))